VatSpace/ui.lua

```diff
@@ -267,6 +267,7 @@ function M.showHelp()
 end
 
 function M.showCredits()
+    gv.showLeaderboards()
```

Comment:
I guessed that this is only a temporary solution. To solve this, I recommend creating a new button in the menu or an icon with e.g. a cup in the bottom corner.